### PR TITLE
Hope this works

### PR DESCRIPTION
**Pull request by VeroxCode**
Added new Speed, Improved Tower

### CODE REVIEW

File: Horion/Module/Modules/AutoGapple.cpp [hunk 1]
```diff
@@ -8,7 +8,7 @@ AutoGapple::~AutoGapple() {
 }
 
 const char* AutoGapple::getModuleName() {
-	return ("AutoGapple");
+	return ("AutoGaple");
```
Comment (richardletshacks) [inline Horion/Module/Modules/AutoGapple.cpp:11]: why?

File: Horion/Module/Modules/Speed.cpp [hunk 2]
```diff
@@ -14,13 +19,44 @@ const char* Speed::getModuleName() {
 void Speed::onTick(C_GameMode* gm) {
 	C_LocalPlayer* localPlayer = g_Data.getLocalPlayer();
 
-	float* speedAdr = reinterpret_cast<float*>(g_Data.getLocalPlayer()->getSpeed() + 0x84);
-	*speedAdr = speed;
+	if (timer == true) {
```
Comment (richardletshacks) [inline Horion/Module/Modules/Speed.cpp:22]: ``== true`` is redundant

File: Horion/Module/Modules/Tower.cpp [hunk 3]
```diff
@@ -3,7 +3,8 @@
 #include "../../DrawUtils.h"
 
 Tower::Tower() : IModule(0x0, Category::WORLD, "Like scaffold, but vertically and a lot faster") {
-	registerFloatSetting("motion", &this->motion, this->motion, 0.3f, 1.f);
+	//registerFloatSetting("motion", &this->motion, this->motion, 0.3f, 1.f);
```
Comment (richardletshacks) [inline Horion/Module/Modules/Tower.cpp:6]: why remove this

File: Horion/Module/Modules/AutoGapple.cpp (hunk 1 above)
Comment (ghost) [reply Horion/Module/Modules/AutoGapple.cpp:11]: LOL

**Pull request closed without merging**